css/impact.css

```diff
@@ -61,7 +24,7 @@ Hero
 	align-items: center;
 }
 
-/* about heading layout and sizing */
+/* impact heading layout and sizing */
```

Comment:
FYI this is moving to global.css via my Events PR

Reply:
I have removed it for now - waiting for events pr for it to be fixed